src/tests/integration/test_duckDB.py

```diff
@@ -1,64 +1,16 @@
 from unittest.mock import Mock
-import logging
 import pytest
 
 import polars as pl
 import pandas as pd
 
-from sql.connection import Connection
+from sql.connection import ConnectionManager
 from sql.warnings import JupySQLDataFramePerformanceWarning
 
 
-@pytest.mark.parametrize(
-    "ip, exp",
-    [
-        (
-            "ip_with_duckDB",
-            "'duckdb.DuckDBPyConnection' object has no attribute "
-            "'set_isolation_level'\n",
-        ),
-        (
-            "ip_with_duckDB_native",
-            "'DBAPISession' object has no attribute 'execution_options'",
-        ),
-    ],
-)
-def test_auto_commit_mode_on(ip, exp, caplog, request):
-    ip = request.getfixturevalue(ip)
-    with caplog.at_level(logging.DEBUG):
-        ip.run_cell("%config SqlMagic.autocommit=True")
-        ip.run_cell("%sql CREATE TABLE weather4 (city VARCHAR,);")
-    assert caplog.record_tuples[0][0] == "root"
-    assert caplog.record_tuples[0][1] == logging.DEBUG
-    assert (
-        "The database driver doesn't support such AUTOCOMMIT"
-        in caplog.record_tuples[0][2]
-    )
-    assert exp in caplog.record_tuples[0][2]
-
-
-@pytest.mark.parametrize(
-    "ip",
-    [
-        ("ip_with_duckDB"),
-        ("ip_with_duckDB_native"),
-    ],
-)
-def test_auto_commit_mode_off(ip, caplog, request):
-    ip = request.getfixturevalue(ip)
-    with caplog.at_level(logging.DEBUG):
-        ip.run_cell("%config SqlMagic.autocommit=False")
-        ip.run_cell("%sql CREATE TABLE weather (city VARCHAR,);")
-    # Check there is no message gets printed
-    assert caplog.record_tuples == []
-    # Check the tables is created
-    tables_out = ip.run_cell("%sql SHOW TABLES;").result
-    assert any("weather" == table[0] for table in tables_out)
-
-
 def test_dbapi_connection_sets_right_dialect(ip_with_duckDB_native):
```

Comment:
Can we add 1 more case when it's not duckdb? (I guess dialect can be `None` or `duckdb`)

Reply:
[fixed](https://github.com/ploomber/jupysql/pull/755/commits/493d8d933f67a319367a71397932822f033cbebe)